packages/datetime/src/datePicker.tsx

```diff
@@ -195,7 +195,7 @@ export class DatePicker extends AbstractComponent<IDatePickerProps, IDatePickerS
     private getDisabledDaysModifier = () => {
         const { dayPickerProps: { disabledDays } } = this.props;
 
-        return disabledDays instanceof Array ? [this.disabledDays, ...disabledDays] : [this.disabledDays, disabledDays];
+        return Array.isArray(disabledDays) ? [this.disabledDays, ...disabledDays] : [this.disabledDays, disabledDays];
```

Comment:
[`Array.prototype.concat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat?v=b) actually handles both cases rather neatly!

`return this.disabledDays.concat(disabledDays);`

Reply:
thats something @cmslewis suggested already. typescript compiler complains when doing this.

Reply:
oh sure, cuz `this.disabledDays` isn't necessarily an array. ok this is fine.